--- webapps/cruisecontrol/main.jsp	2009-10-19 13:42:02.000000000 +0200
+++ webapps/cruisecontrol/main.jsp	2009-10-19 13:44:48.000000000 +0200
@@ -130,6 +130,12 @@
             </cruisecontrol:tab>
             
             <%-- phpUnderControl 8 --%>
+            
+            <!-- Padawan -->
+            <cruisecontrol:tab name="padawan" label="Padawan">
+              <cruisecontrol:xsl xslFile="/xsl/padawan_details.xsl"/>
+            </cruisecontrol:tab>
+
 
           </td>
         </tr>
